include/pg_tmgr.h

```diff
@@ -0,0 +1,29 @@
+/*-------------------------------------------------------------------------
+ *
+ * include/pg_tmgr.h
```

Comment:
maybe name it transaction_manager.h and transaction_manager.c, the pg_ prefix suggests this is for PostgreSQL in general.